### PR TITLE
Add option to disable the stdout log timestamp

### PR DESCRIPTION
**Pull request by pieterlexis**
And add it to the unit files. This fixes the "double timestamps" issue.

### CODE REVIEW

**Comment by Habbie** [comment]
The travis failures appear to be real. rebase perhaps?

File: pdns/logger.cc [hunk 1]
```diff
@@ -47,14 +47,17 @@ void Logger::log(const string &msg, Urgency u)
...
     char buffer[50];
-    strftime(buffer,sizeof(buffer),"%b %d %H:%M:%S ", &tm);
+    buffer[0] = '\0';
+    if(!d_disableTimeStamps) {
+      struct tm tm;
+      time_t t;
+      time(&t);
+      tm=*localtime(&t);
+
+      strftime(buffer,sizeof(buffer),"%b %d %H:%M:%S ", &tm);
```
Comment (Habbie) [inline pdns/logger.cc]: Unrelated to this patch but we should, theoretically, be checking the return value of `strftime`.

File: pdns/logger.hh [hunk 2]
```diff
@@ -60,6 +60,10 @@ public:
     d_disableSyslog = d;
   }
 
+  void disableTimeStamps(bool d) {
+    d_disableTimeStamps = d;
```
Comment (Habbie) [inline pdns/logger.hh]: the setting is singular but this bool and method are plural, looks a bit weird. I suggest making the setting plural.

**Comment by Habbie** [comment]
Can you rebase and look at the nits? Thanks!

**Comment by pieterlexis** [comment]
closing for now, this fails to build on travis but works locally

**Pull request closed without merging**